.ci/.env.prd

```diff
@@ -32,6 +32,6 @@ REACT_APP_MARKETPLACE_GRAPH_URL=https://api.thegraph.com/subgraphs/name/decentra
 REACT_APP_IPFS_URL=https://ipfs.infura.io:5001/api/v0/add?pin=false
 REACT_APP_LAND_POSITION_URL=https://play.decentraland.org?position=
 REACT_APP_CLAIM_NAME_URL=https://avatars.decentraland.org
-REACT_APP_FF_WEARABLES=
 REACT_APP_FF_ENS=1
+REACT_APP_FF_WEARABLES=
```

Comment:
should we need to add `REACT_APP_FF_WEARABLES_PUBLISH=` as well?

Reply:
I didn't because the flag shouldn't reach prod, but looking at it, I'll add it for completeness sake